public/static/pathways/neoadjuvant_pathway.json

```diff
@@ -2,7 +2,7 @@
     "name": "Breast Cancer: Neoadjuvant Chemotherapy with Surgery ",
     "description": "Pathway for the Early Stage HER2+ Breast Cancer Pathway for neoadjuvant chemotherapy with surgery.",
     "library": "mCODE_Library.cql",
-    "criteria": [
+    "precondition": [
```

Comment:
Because this is a list of objects this could probably be renamed to the plural "preconditions". But not a showstopper

Reply:
I changed my mind - please change this so it's plural, that way it's consistent with `nodes`